src/Components/Artist/MarketInsights.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface MarketInsightsProps extends RelayProps, React.HTMLProps<MarketInsights> {}
+
+const Categories = {
+  "blue-chip": "Blue Chip",
+  "top-established": "Top Established",
+  "top-emerging": "Top Emerging",
+}
+
+export class MarketInsights extends React.Component<MarketInsightsProps, null> {
+  renderGalleryRepresentation() {
+    const { highlights } = this.props.artist
+    const { partners } = highlights
+    if (partners && partners.edges && partners.edges.length > 0) {
+      const partner = partners.edges[0].node
+      const { categories } = partner
+      let category
+      Object.keys(Categories).forEach(key => {
+        categories.forEach(partnerCategory => {
+          if (partnerCategory.id === key) {
+            category = Categories[key]
+          }
+        })
+      })
+      return (
+        <div>
+          Represented by {partner.name}
+          <br />
+          This is a {category} type of partner.
+        </div>
+      )
+    }
+  }
+
+  renderAuctionHighlight() {
+    if (!this.props.artist.auctionResults || this.props.artist.auctionResults.edges.length < 1) {
+      return null
+    }
+    const topAuctionResult = this.props.artist.auctionResults.edges[0].node
+
+    return (
+      <div>
+        Top auction result is {topAuctionResult.price_realized} at {topAuctionResult.organization}
+      </div>
+    )
+  }
+
+  renderPermanentCollection() {
+    const { collections } = this.props.artist
+    if (collections && collections.length > 0) {
+      return (
+        <div>
+          Collected by major museums
+          <br />
+          {collections.join(", ")}
+        </div>
+      )
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        {this.renderGalleryRepresentation()}
+        <br />
+        {this.renderAuctionHighlight()}
+        <br />
+        {this.renderPermanentCollection()}
+      </div>
+    )
+  }
+}
+
+export default createFragmentContainer(
+  MarketInsights,
+  graphql.experimental`
```

Comment:
What's this about?

Reply:
Relay had some custom GraphQL directives that they weren’t yet committed to landing in the final API (such as fragment `@arguments`), but in newer Relay versions this is no longer ‘experimental’ nor does it seem like this fragment even uses any of these directives, so it can be changed to just `GraphQL`.

Reply:
Oh cool, I'll undo.